eng/templates/stages/release.yml

```diff
@@ -148,6 +152,33 @@ stages:
             draft='--draft'
           fi
 
+          git_user=dotnet-bot
+          git_email=dotnet-bot@microsoft.com
+
+          git config --global user.name "$git_user"
+          git config --global user.email "$git_email"
+
+          vmr_url=$(echo $vmrPublicUrl | sed "s,https://.*@,https://$(git_user):${GH_TOKEN}@,g")
+
+          git fetch $vmr_url tag $releaseTag --no-tags
+          if [[ $? = 0 ]]; then
+            tagged_commit=$(git rev-list -1 refs/tags/$releaseTag)
```

Comment:
Why not this?
```suggestion
            tagged_commit=$(git rev-parse "refs/tags/$releaseTag")
```

Reply:
According to [this](https://stackoverflow.com/a/1862542) thread there is a small difference between the two for annotated tags as `rev-parse` shows the SHA1 of the tag and not the commit, leading to a potential diff depending on the type of tag used.

Probably not a big difference in our case, can use `rev-parse` if needed

Reply:
Ah, yeah this would bite us if ever created an annotated one. Thanks!